src/Fixer/Operator/TernaryToElvisOperatorFixer.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Operator;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Gabriel Caruso <carusogabriel34@gmail.com>
+ */
+final class TernaryToElvisOperatorFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefinition()
+    {
+        return new FixerDefinition(
+            'Converts a ternary operation with first and second operands identicals to a elvis operatorion (?:).',
+            [
+                new CodeSample(
+                    "<?php \$foo = \$bar ? \$bar : baz;\n"
+                ),
+            ]
+        );
+    }
+
+    public function isCandidate(Tokens $tokens)
```

Comment:
not sure how we go in most of the cases, but I would add `inheritdoc`

Reply:
Sorry, missed that one